### DISCUSSION TITLE
ThisIsWin11 1.4.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/ThisIsWin11#199] [Announcements]
### [builtbybel.com](https://www.builtbybel.com) has been updated

- Sign of Life! **The project is alive** 🚀 Had a busy schedule the past couple of months.....I'm slowly waking up again
- Titlebar matches Windows now and has Mica behind it https://github.com/builtbybel/ThisIsWin11/pull/175
- [ThisIsWin11](https://github.com/builtbybel/ThisIsWin11) and [LoveWindowsAgain App](https://github.com/builtbybel/LoveWindowsAgain) have now been merged into one package. 
_The latter application builds on the code of the former and partly vice versa. If you want to do it fast I recommend the LoveWindowsAgain App, for in-depth customizations you can use the ThisIsWin11 monster_
 -Fix some links
- Minor fixes

![kT7j5ZxE9A](https://user-images.githubusercontent.com/57478606/185175165-ef120131-5de7-443f-b5bb-bc36f96e63cb.png)

**The LoveWindowsAgain app is only rolled out in the complete TIW11.zip package and is not distributed via the automatic update function.**

_Use the LoveWindowsAgain asset file, to get the standalone version of [LoveWindowsAgain ](https://github.com/builtbybel/LoveWindowsAgain)._
_Please always download the TIW11.zip file as it contains the complete application package._

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/ThisIsWin11/releases/tag/1.4.0'>ThisIsWin11 1.4.0</a>.</em>

**Comment by rossrankin** [comment]
Rather than having it fix all automatically, it should have checkboxes next to each recommendation so you can pick and choose.

**Comment by wacieni** [comment]
DIscovered this thanks to Craft Computing TG recommends
